src/main/java/org/spongepowered/common/data/ImmutableDataCachingUtil.java

```diff
@@ -70,15 +70,14 @@ private ImmutableDataCachingUtil() {}
         try {
             // Let's get the key
             return (T) (Object) ImmutableDataCachingUtil.manipulatorCache
-                .get(key,
```

Comment:
Don't touch the `data` package. I've got [changes](https://github.com/SpongePowered/SpongeCommon/compare/feature/data) that are still incubating.